apps/api/src/server/applications/application/documents/__tests__/store-document-metadata.test.js

```diff
@@ -1,26 +1,44 @@
 import { jest } from '@jest/globals';
-const { databaseConnector } = await import('../../../../utils/database-connector.js');
-
-import { request } from '../../../../app-test.js';
+import { request } from '#app-test';
 import { applicationStates } from '../../../state-machine/application.machine.js';
+import { EventType } from '@pins/event-client';
+import { NSIP_DOCUMENT } from '#infrastructure/topics.js';
+
+const { databaseConnector } = await import('#utils/database-connector.js');
+const { eventClient } = await import('#infrastructure/event-client.js');
+
+const dateDocCreated = '2022-01-01T11:59:38.129Z';
+const dateDocLastModified = '2024-02-05T11:59:38.129Z';
```

Comment:
Should this be in the past or do we want to call `new Date()` to get the current datetime?

Reply:
As it is just test data I was not sure if it was important to get latest date, I just happened to update it to a new date as I was updating the tests.  I can apply this if you think it is necessary?

Reply:
Cool that's fine, I was just curious as I saw it was a recent date.